main.py

```diff
@@ -464,18 +465,18 @@ def history(self):
     def help(self):
         help_text = [
             _("start - Description"),
-            _("add - Add song"),
+            _("add - Add song by full path"),
             _("sample - Randomly list some songs"),
-            _("status - DJ Status"),
-            _("stats - DJ stats"),
+            _("status - Now playing"),
+            _("stats - Songs quantity"),
             _("order - Order a song"),
             _("searchorder - Search and order first match song"),
             _("next - Skip current song"),
             _("history - Order history"),
             _("search - Search song"),
             _("searchadd - Search and add first match song"),
             _("playlist - Show current playlist"),
-            _("list - List files"),
+            _("list - List files (don't do this)"),
```

Comment:
This is somewhat irrelevant to your change, also, you might consider to make this command to be superuser only.